Core/src/org/sleuthkit/autopsy/discovery/DiscoveryEventUtils.java

```diff
@@ -332,4 +373,33 @@ int getGroupSize() {
         }
 
     }
+
+    /**
+     * Event to signal that the visibility of the Details area should change.
+     */
+    static class DetailsVisibleEvent {
+
+        private final boolean showDetailsArea;
+
+        /**
+         * Construct a new DetailsVisibleEvent.
+         *
+         * @param isVisible True if the details area should be visible, false
+         *                  otherwise.
+         */
+        DetailsVisibleEvent(boolean isVisible) {
+            showDetailsArea = isVisible;
+            //no arg constructor
```

Comment:
Again, it seems like there's an argument given...

Reply:
yeah woops